src/views/pam/RiskDetect/AccountCheckTaskList.vue

```diff
@@ -31,37 +31,30 @@ export default {
         },
         columnsMeta: {
           name: {
-            formatter: DetailFormatter,
-            formatterArgs: {
-              route: 'AccountCheckDetail'
-            }
+            formatter: (row) => row.name
           },
           assets: {
             formatter: AmountFormatter,
             formatterArgs: {
               async: false,
-              getRoute({ row }) {
-                return {
-                  name: 'AssetDetail',
-                  params: {
-                    id: row.id
-                  }
-                }
-              }
+              drawer: true,
+              getTitle: ({ row }) => row.assets[0].name,
+              getRoute: ({ row }) => ({
+                name: 'AssetDetail',
+                params: { id: row.assets[0].id }
+              })
             }
           },
           nodes: {
             formatter: AmountFormatter,
             formatterArgs: {
               async: false,
-              getRoute({ row }) {
-                return {
-                  name: 'AssetDetail',
-                  params: {
-                    id: row.id
-                  }
-                }
-              }
+              drawer: true,
+              getRoute: ({ row }) => ({
+                name: 'AssetDetail',
+                query: { tab: 'Basic' },
+                params: { id: row.assets[0].id }
+              })
             }
           },
           secret_strategy: {
```

Comment:
This is the original version of the provided JSON. There are no known irregularities but some options to consider:

```
export default {
     columnsMeta: {
        name: {
           formatter: DetailFormatter,
          formatterArgs: {
            route: 'AccountCheckDetail'
    }
```